test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -657,18 +663,33 @@ class TestDapp {
 
   /**
    * Sign a message with the signTypedDataV4 method.
+   *
+   * @param confirmationRedesign - Indicates whether the redesigned signature confirmation flow is used. Defaults to false.
    */
-  async signTypedDataV4() {
+  async signTypedDataV4(confirmationRedesign: boolean = false) {
```

Comment:
Keep the `confirmationRedesign` as a parameter as a temporary solution, will do the refacto once the toggle is removed, as planned in [MetaMask-planning issue #303](https://github.com/MetaMask/MetaMask-planning/issues/303).